CHNS Free Energy Postprocessing (#1095)
lethe-cfd/lethe: doc/source/parameters/cfd/post_processing.rst

    # Other Cahn-Hilliard postprocessing
    set calculate phase statistics       = false
    set phase statistics name            = phase_statistics

>     set calculate phase energy           = true

Reviewer: True by default? It should be false, no?

Reviewer: What do you think about removing the word "calculate" from the parameters? (can be another pr)

Author: should definitely be false, you're right. For the calculate I think I'll leave it to another PR

* ``mass conservation name``: name of the output file containing the mass of both fluids for VOF simulations. The default file name is ``mass_conservation_information``.
  

> * ``calculate phase statistics``: outputs phase statistics from the solution of the Cahn-Hilliard equations, including minimum, maximum, average, integral of the phase order parameter. Also includes the volume of each phase. This works only with the :doc:`cahn_hilliard` solver.

Reviewer: ```suggestion
* ``calculate phase statistics``: outputs phase statistics from the solution of the Cahn-Hilliard equations, including minimum, maximum, average, and integral of the phase order parameter. The volume of each phase is also computed. This works only with the :doc:`cahn_hilliard` solver.
```

Reviewer: " This works only with the :doc:`cahn_hilliard` solver." can be a warning

Reviewer: I would rephrase it to: outputs Cahn-Hilliard phase statistics, including .............

* ``phase statistics name``: name of the output file containing phase order parameter statistics from Cahn-Hilliard simulations. The default file name is ``phase_statistics``.

> * ``calculate phase energy``:  outputs phase energies from the solution of the Cahn-Hilliard equations : bulk energy, interface energy and total energy. This works only with the :doc:`cahn_hilliard` solver. The energies are computed as follow:

Reviewer: ```suggestion
* ``calculate phase energy``:  outputs phase energies from the solution of the Cahn-Hilliard equations, including: bulk energy, interface energy, and total energy. This works only with the :doc:`cahn_hilliard` solver. The energies are computed as:
```

Reviewer: Same comment about the warning

  .. math::

     E_{bulk} = \int_{\Omega} (1-\phi^2)^2 \mathrm{d}\Omega 
      
  .. math::

     E_{interface} = \int_{\Omega} 0.5\epsilon^2|\nabla \phi |^2 \mathrm{d}\Omega 
      
  .. math::

     E_{total} = E_{bulk} + E_{interface}  
    

>   where :math:`\epsilon` is the numerical interface thickness. Note that these energies are not homogeneous to physical energies, but they give nonetheless a good insight on the system's evolution.

Reviewer: ```suggestion
  where :math:`\epsilon` is the numerical interface thickness. Note that these energies are not homogeneous to physical energies. Nonetheless, they are a convenient way to track the system's evolution.
```

> * ``phase energy name``: name of the output file containing phase energies from Cahn-Hilliard simulations. The default file name is ``phase_energy``.

Reviewer: Is there an extension to this file?
Where is it stored? at the output folder?
Could you add this information here?
